layouts/default.vue

```diff
@@ -71,7 +71,7 @@
                                   v-bind="attrs"
                                   v-on="on"
                                   small
-                                  color="green"
+                                  :color="faucet_status_color"
```

Comment:
I believe this is not the root cause of the status not working, this should be something happening in the api call to the backend. 
https://github.com/canow-co/faucet-ui/blob/83152d0f25728a39432845acace1dd2ffa8a1ee0/api/status/index.js#L9

Reply:
@hd5666 it's just the color update (ui)